src/main/java/seedu/address/model/person/Nric.java

```diff
@@ -15,7 +15,7 @@ public class Nric {
      * Should be an alphanumeric string with 4 characters, with the first 3
      * being numbers and the last being a letter.
      */
-    public static final String VALIDATION_REGEX = "^\\d{3}[A-Z]$";
+    public static final String VALIDATION_REGEX = "^\\d{3}[A-Za-z]$";
```

Comment:
This will allow the user to input case-insensitive alphabets for their NRIC, great!